manifests/t/TexasInstruments/TI-SmartView/TI-84Plus/5.6.0.2092/TexasInstruments.TI-SmartView.TI-84Plus.locale.en-US.yaml

```
@@ -0,0 +1,28 @@
+# Created using wingetcreate 0.4.3.1
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.0.0.schema.json
+
+PackageIdentifier: TexasInstruments.TI-SmartView.TI-84Plus
+PackageVersion: 5.6.0.2092
+PackageLocale: en-US
+Publisher: Texas Instruments Inc.
+PublisherUrl: https://education.ti.com
+PublisherSupportUrl: https://education.ti.com/en/customer-support
+PrivacyUrl: https://www.ti.com/legal/terms-conditions/privacy-policy.html
+Author: Texas Instruments Inc.
+PackageName: 'TI-SmartView CE Emulator Software for the TI-84 Plus Family '

```

```suggestion
PackageName: TI-SmartView CE for the TI-84 Plus Family
```
This must match Control Panel.